clang/lib/CIR/Dialect/Transforms/LoweringPrepare.cpp

```diff
@@ -127,6 +127,13 @@ void LoweringPreparePass::lowerGlobalOp(GlobalOp op) {
     // Add a function call to the variable initialization function.
     dynamicInitializers.push_back(f);
   }
+
+  auto &dtorRegion = op.getDtorRegion();
+  if (!dtorRegion.empty()) {
+    // TODO: handle dtor region
```

Comment:
I'd prefer a llvm_unreachable here for the general approach for incremental work, but this is fine this time around since you're adding support next.